tests/setup.cfg

```diff
@@ -1,26 +0,0 @@
-[flake8]
-
-max-line-length = 160
-
-ignore = \
-    # Rules that flake8 ignores by default (we have to put these in our `ignore`
-    # setting otherwise they would be un-ignored).
-    E121,E123,E126,E226,E24,E704,W503 \
-
-    # Lint warnings that are specifically disabled for tests:
-    #   E241 - multiple spaces after ','
-    #   E722 - do not use bare except
-    #
-    #   N802 - function name should be lowercase
-    #   N803 - argument name should be lowercase
-    #   N806 - variable name should be lowercase
```

Comment:
Removed the whole file and we will rely on the setup.py on the project's root. I'm surprised this hasn't caused the current lint make target to fail :shrug: